### DISCUSSION TITLE
Open Launcher in Tiled Mode

### DISCUSSION THREAD

**Discussion by alliqmahbuby** [YaLTeR/niri#852] [Q&A] [answered]
Is it possible to run app launcher in tiled mode instead of popup?

![image](https://github.com/user-attachments/assets/e2dd8b71-f42e-4f64-a9f3-27229c797cf1)

**Comment by YaLTeR** [comment, marked as answer]
If the launcher can open as a normal window, rather than layer-shell, then that will do the trick. Otherwise no, layer-shell surfaces are not part of the normal layout.
